pnpm-lock.yaml

```diff
@@ -5618,6 +5621,18 @@ packages:
       react-dom: 17.0.2(react@17.0.2)
     dev: false
 
+  /@radix-ui/react-primitive@1.0.2(react-dom@17.0.2)(react@17.0.2):
```

Comment:
The third instance of react-primitive. We will need to bump radix packages.

Reply:
lets bump

Reply:
separate pr for bumping or same?